File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/14.java
public class AFNSem11Ou00 {
    public static boolean sem11Ou00(String s) {
        return !s.contains("11") && !s.contains("00");
    }

    public static void main(String[] args) {
        System.out.println(sem11Ou00("1010"));  // True
        System.out.println(sem11Ou00("1100"));  // False
    }
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/14.go
package main
import "fmt"
import "strings"

func sem11Ou00(s string) bool {
    return !strings.Contains(s, "11") && !strings.Contains(s, "00")
}

func main() {
    fmt.Println(sem11Ou00("1010"))  // True
    fmt.Println(sem11Ou00("1100"))  // False
}
